apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -129,27 +129,28 @@ const ExpandedCurriculumCatalogCard = ({
                 </div>
                 <div className={style.linksContainer}>
                   <div className={style.resourcesContainer}>
-                    {Object.keys(availableResources).length > 0 && (
-                      <div>
-                        <Heading4 visualAppearance="heading-xs">
-                          {i18n.availableResources()}
-                        </Heading4>
-                        <hr className={style.thickDivider} />
-                        {resoucesOrder.map(
-                          resource =>
-                            availableResources[resource] && (
-                              <div>
-                                <BodyTwoText>
-                                  {translatedAvailableResources[resource]}{' '}
-                                </BodyTwoText>
-                                {displayDivider() && (
-                                  <hr className={style.horizontalDivider} />
-                                )}
-                              </div>
-                            )
-                        )}
-                      </div>
-                    )}
+                    {availableResources &&
+                      Object.keys(availableResources).length > 0 && (
```

Comment:
I think these two lines do the same thing so you probably just need `availableResources && ...`